release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -368,6 +379,55 @@ Red Hat Virtualization (RHV) will be deprecated in an upcoming release of {produ
 [id="ocp-4-12-removed-features"]
 === Removed features
 
+[id="ocp-4-12-removed-kube-1-25-apis"]
+==== Beta APIs removed from Kubernetes 1.25
+
+Kubernetes 1.25 removed the following deprecated APIs. Migrate manifests and API clients to use the appropriate API version. For more information about migrating removed APIs, see the link:https://kubernetes.io/docs/reference/using-api/deprecation-guide/#v1-25[Kubernetes documentation].
```

Comment:
@bergerhoffer `Kubernetes 1.25 removed the following deprecated APIs. Migrate manifests and API clients to use the appropriate API version.` this sentence reads a little bit oddly. Can we say?
`Kubernetes 1.25 removed the following deprecated APIs, so using the appropriate API version to migrate manifests and API clients are required.`  Just my suggestion.